script/files/boot/dts/overlays/poweroff-pins.dts

```diff
@@ -0,0 +1,26 @@
+/dts-v1/;
+
+/ {
+	compatible = "brcm,bcm2708";
+
+	fragment@0 {
+		target-path = "/";
+		__overlay__ {
+			poweroff_gpio: poweroff_gpio {
+				pinctrl-names = "default";
+				pinctrl-0 = <&poweroff_pins>;
+			};
+		};
+	};
+
+	fragment@1 {
+		target = <&gpio>;
+		__overlay__ {
+			poweroff_pins: poweroff_pins {
+				brcm,pins = <23 24>;
```

Comment:
I still have yet to boot of the image with this loaded, so we'll see. Is there a way to test that a pin was configured correctly?